templates/aws-tap-entrypoint-existing-vpc.template.yaml

```diff
@@ -501,9 +494,8 @@ Parameters:
   TAPVersion:
     Type: String
     Description: >-
-      Version of TAP to deploy. The selected version must also support the
-      selected Kubernetes version (KubernetesVersion). For more information,
-      refer to Kubernetes cluster requirements
+      Version of TAP to deploy. For more information, refer to Kubernetes
```

Comment:
Since we only allow one version of EKS and one version of TAP, referencing the cluster requirements adds more confusion than clarification

Reply:
True. Which is why I asked those 2 questions in slack:
- do we even want to expose the EKS/l8s version to the users?
- de we even want to expose the TAP version to the users?

If we settle on both or either one being "no" those descriptions would go away (or at least not be exposed to the users).